ingestion-beam/src/test/java/com/mozilla/telemetry/transforms/PubsubLiteCompatTest.java

```diff
@@ -12,7 +12,6 @@
 import com.mozilla.telemetry.ingestion.core.util.IOFunction;
 import com.mozilla.telemetry.options.OutputFileFormat;
 import com.mozilla.telemetry.util.Json;
-import com.mozilla.telemetry.util.PubsubWithNullsJsonCoder;
```

Comment:
special coder is no longer needed, because it was only needed to code messages with null payload